petastorm/spark/spark_dataset_converter.py

```diff
@@ -93,15 +107,21 @@ def make_spark_converter(df, cache_dir=None, row_group_size=ROW_GROUP_SIZE):
                       Default None, it will fallback to the spark config
                       "spark.petastorm.converter.default.cache.dir".
                       If the spark config is empty, it will fallback to DEFAULT_CACHE_DIR.
-    :param row_group_size: An int denoting the number of bytes in a parquet row group.
+    :param compression: Specify compression type. Default None.
```

Comment:
Shall we change it to `A string specifying the compression type.` It would also be helpful if we indicate what values can be chosen from.